doc/source/examples/multiphysics/capillary-wave/capillary-wave.rst

```diff
@@ -0,0 +1,379 @@
+
+.. code-block:: text
+
+    subsection mesh adaptation
+      set type                     = kelly
+      set variable                 = phase
+      set fraction type            = fraction
+      set max refinement level     = 5
+      set min refinement level     = 3
+      set frequency                = 1
+      set fraction refinement      = 0.95
+      set fraction coarsening      = 0.05
+      set initial refinement steps = 4
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~
+
+In the ``physical properties`` subsection, we define the fluids such that both fluids have the same properties. We set the ``density`` to :math:`1` and the ``kinematic viscosity`` to :math:`5 \times 10^{-6}`. A ``fluid-fluid`` type of material interaction is also defined to specify the ``surface tension model``. In this case, it is set to ``constant`` with the ``surface tension coefficient`` set to :math:`0.01`.
+
+.. code-block:: text
+
+    subsection physical properties
+      set number of fluids = 2
+      subsection fluid 1
+        set density             = 1
+        set kinematic viscosity = 5e-6
+      end
+      subsection fluid 0
+        set density             = 1
+        set kinematic viscosity = 5e-6
+      end
+      set number of material interactions = 1
+      subsection material interaction 0
+        set type = fluid-fluid
+        subsection fluid-fluid interaction
+          set first fluid id              = 0
+          set second fluid id             = 1
+          set surface tension model       = constant
+          set surface tension coefficient = 0.01
+        end
+      end
+    end
+
+
+-----------------------
+Running the Simulation
+-----------------------
+
+We can call ``lethe-fluid`` for each time step value. For :math:`\Delta t \approx 0.95\Delta t_\sigma`, this can be done by invoking the following command:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 4 lethe-fluid capillary-wave-TSM-0_95.prm
+
+to run the simulation using four CPU cores. Feel free to use more CPU cores.
+
+.. warning:: 
+    Make sure to compile Lethe in `Release` mode and run in parallel using mpirun.
+    This simulation takes :math:`\sim \, 35` minutes for :math:`\Delta t\approx 0.95\Delta t_\sigma` and decreases to :math:`\sim \, 3` minutes for :math:`\Delta t\approx 20\Delta t_\sigma` on :math:`4` processes.
+
+.. tip::
+  In order to calculate the capillary time-step constraint and the simulation end time, a small Python script is provided, you may run it using:
+
+  .. code-block:: text
+    :class: copy-button
+
+    python3 capillary-wave-calculation.py calculations.output
+
+  ``calculations.output`` is the file where the results will be saved. If you omit this argument, results will simply be displayed in the terminal window.
+
+  .. attention::
```

Comment:
This attention section is nested in the above tip section, but I don’t think that’s intended.

Reply:
Actually it is, since it talks about ``capillary-wave-calculation.py`` the script mentioned in the tip.